### PR TITLE
feat/change according to new schema

### PR DESCRIPTION
**Pull request by new5558**
- Change from end to finish because end is PostgresSQL reserved word.
- add new message `FacilityRequestWithFacilityInfo `
- Change interface in `GetFacilityRequestStatusus`

### CODE REVIEW

File: proto/hts/facility/service.proto [hunk 1]
```diff
@@ -21,7 +21,7 @@ message FacilityRequest {
     int64 is_approved = 4;
     int64 reject_reason = 5;
     google.protobuf.Timestamp start = 6;
-    google.protobuf.Timestamp end = 7;
```
Comment (boompikachu) [inline proto/hts/facility/service.proto:24]: Why change to finish?

File: proto/hts/facility/service.proto [hunk 2]
```diff
@@ -52,10 +68,14 @@ message GetFacilityRequestListResponse {
     repeated FacilityRequest requests  = 1; 
 }
 
-message GetFacilityRequestStatusRequest {
+message GetFacilityRequestStatususRequest {
```
Comment (boompikachu) [inline proto/hts/facility/service.proto:71]: Statusus?

Comment (new5558) [reply proto/hts/facility/service.proto:71]: return array so plural noun

File: proto/hts/facility/service.proto (hunk 1 above)
Comment (new5558) [reply proto/hts/facility/service.proto:24]: end is restricted by database

File: proto/hts/facility/service.proto (hunk 2 above)
Comment (boompikachu) [reply proto/hts/facility/service.proto:71]: plural noun: statuses

Comment (new5558) [reply proto/hts/facility/service.proto]: NVM. Changed to `GetFacilityRequestsStatus`

**Review by boompikachu — APPROVED** [review]
LGTM

**Pull request merged**